### ISSUE TITLE
Add config value for refresh rate

### ISSUE THREAD

**Issue by jez9999** [joncampbell123/dosbox-x#5224]
### Is your feature request related to a problem? Please describe.

You can change the refresh rate from the Dosbox-X dropdown menus "Video | Adjust refresh rate", but I don't see a way to do it from config.

### What you want

Allow refresh rate to be set from config.

### Have you checked that no similar feature request(s) exist?

- [X] I have searched and didn't find any similar feature request.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by maron2000** [comment]
Isn't `vsyncrate` the option you're mentioning?
It's prettey easy to find in both .conf file and the configuration tool.

**Comment by jez9999** [comment]
I'm not sure it is.  I have it set to 75 in config, but in the UI the "refresh rate" is always zero until I set it (setting it to 60 fixes the graphics for Commander Keen 4).

**Comment by maron2000** [comment]
Seems `forcerate` is the option you want, found in dosbox-x.reference.full.conf
```
forcerate: Force the VGA framerate to a specific value(ntsc, pal, or specific hz), no matter what
```

**Comment by jez9999** [comment]
Yeah that does seem to work.  Maybe it should be in the config file commented out, and some how easier to match up with the UI option "Adjust refresh rate".
